metals/src/main/scala/scala/meta/internal/metals/FormattingProvider.scala

```diff
@@ -553,11 +553,9 @@ object FormattingProvider {
 
   }
 
-  def newScalafmt(respectVersion: Boolean = true): Scalafmt =
+  def newScalafmt(): Scalafmt =
     Scalafmt
       .create(this.getClass.getClassLoader)
       .withReporter(EmptyScalafmtReporter)
-      .withDefaultVersion(BuildInfo.scalafmtVersion)
-      .withRespectVersion(respectVersion)
```

Comment:
So these are both deprecated now, so I instead went the route where in `DownloadDependencies` I write a basic config file so that it's still able to pull down scalafmt and run once. When testing locally with a config that doesn't have a scalafmt version the behavior is still the same of prompting the user for a version and then addin git.